app/Models/OutcomeData.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use DB;
+class OutcomeData extends Model
+{
+    /**
+     * check the submitted statement is logged
+     *
+     * @var int
+     * @var int
+     * @var string
+     * 
+     */
+    public static function isSubmitted($actor_id, $activity_id, $submission_id) {
+        $result = DB::select("select * from outcome_data where user_id = ? AND submission_id = ? AND assignment_id = ? AND verb= ? order by page_order desc", [$actor_id, $submission_id, $activity_id,'submitted-curriki']);
+        if(count($result) > 0)
+            return true;
+        else
+            return false;
+    }
+
```

Comment:
Function opening brace should be on the new line
publich function name()
{
}

And if else should have opening/closing brackets.

Reply:
done